src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,8 +5,7 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
-    public Bulldozer() {
-    }
+    private int year;
```

Comment:
The field 'year' should have an access modifier to encapsulate the field properly. Consider using 'private' to restrict direct access.